### ISSUE TITLE
Bring the stuff in the storage room to an accessable spot

### ISSUE THREAD

**Issue by NitramLegov** [comakingspace/do-something#73]
Right now, there is quite a bit of stuff in the storage room which needs to be ordered. This should get back into the common area in order to make it accessable

**Comment by emka** [comment]
This is actually several steps that should happen simultaneously:
1. Inventory each item when taking it out (hand-written list: item, owner, location; to be digitized later)
2. Find a suitable spot and build a holder if required
3. Document the tool in the wiki using our existing templates and categories

**Comment by NitramLegov** [comment]
Sorry, I just opened the issue real quick during the plenum.
The thought was to first move everything "as is" to a storage place where members can access it ( @beanobeano mentioned the to-be-lasercutter-room). Once this happened, members can start helping with checking and inventorizing tools in there (with the steps you just mentioned) :)

**Comment by LukasFreeze** [comment]
I would not like to have unchecked equipment out in the common room (or next to it, doesn't make a big difference): we've worked hard the last few months and are finally close to having only checked & documented equipment out in the open, if we bring a bunch of stuff out at once now it will do no good. As a supervisor I much prefer having new equipment in a spot that only members can access - the ladder shelf - and as one of the active wiki contributors I prefer it in reasonable "portions" - as much as fits under the ladder shelf...

If there are tools under that shelf that are not making progress I'm not against rotating them back into storage of course to make room for something new, but all in all I think we should concentrate on making the tools that we already started with usable (and then easier to use, e.g. collect project ideas and other useful info for beginners).
